modules/ROOT/pages/manage-api-health-checks.adoc

```diff
@@ -0,0 +1,37 @@
+= Managing API Health Checks
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+API health checks enable you to check connection attempts to a specific endpoint. When you configure the health check, specify a specific status code that indicates a healthy connection to the endpoint. If the endpoint connection attempt gives a status that violates the set parameters, an email address you specify receives an email noting the failed health check.
```

Comment:
```suggestion
API health checks enable you to check connection attempts to a specific endpoint. When you configure the health check, specify a specific status code that indicates a healthy connection to the endpoint. If the endpoint connection attempt gives a status that violates the set parameters, an email address you specify receives an email noting the failed health check. Connection attempts for health checks run every 15 minutes.
```

Reply:
Suggestion: Health check is run every 15 minute to validate endpoint connection according to the configuration.